.github/workflows/JOB_tests.yml

```
@@ -12,6 +12,12 @@ jobs:
       matrix:
         os: [ubuntu-latest, macos-latest, windows-latest]
         python-version: ["3.8", "3.9", "3.10", "3.11"]
+        exclude:
+          - os: macos-latest
+            version: "3.8"
+          - os: macos-latest
+            version: "3.9"
+
```

You're right that 3.8 and 3.9 versions are not available on `macos-latest` runners (which now points to M1), but they're available in the pre-M1 runners (`macos-13`).

python/peps has a nice workaround to it - https://github.com/python/peps/pull/3763/files

Adding include blocks for 3.8 and 3.9 (which might do the trick) 
```suggestion
        exclude:
          - os: macos-latest
            version: "3.8"
          - os: macos-latest
            version: "3.9"
        include:
          - os: macos-13
            version: "3.8"
          - os: macos-13
            version: "3.9"
```